src/addresses.h

```diff
@@ -374,6 +374,8 @@
 #define RCT2_ADDRESS_NEXT_WEATHER_GLOOM				0x013CA751
 #define RCT2_ADDRESS_CURRENT_RAIN_LEVEL				0x013CA752
 #define RCT2_ADDRESS_NEXT_RAIN_LEVEL				0x013CA753
+#define RCT2_ADDRESS_CURRENT_SNOW_LEVEL				0x013CA755
+#define RCT2_ADDRESS_NEXT_SNOW_LEVEL				0x013CA756
```

Comment:
Because... if you look at the address of RCT2_ADDRESS_NEWS_ITEM_LIST, it overlaps your new variables. You cannot insert brand new variables in the region of the original RCT2 data sector!